src/content.js

```diff
@@ -551,6 +551,25 @@ function init() {
 			event.preventDefault();
 			indentInput(event.target);
 			return false;
+		} else if (event.which === 13 && event.shiftKey && (event.ctrlKey || event.metaKey)) {
+			const singleCommentButton = event.target.closest('.js-inline-comment-form')
+					.querySelector('.review-simple-reply-button');
+
+			if (singleCommentButton) {
+				event.preventDefault();
+				singleCommentButton.click();
+				return false;
+			}
+		} else if (event.which === 27) {
+			const form = event.target.closest('.js-inline-comment-form');
+			const textarea = form.querySelector('.review-simple-reply-button');
+			const cancelButton = form.querySelector('.js-hide-inline-comment-form');
+
+			if (textarea && textarea.value !== '' && cancelButton) {
```

Comment:
I don’t think we need to check for the content here. If the user presses ESC, they should get the regular confirmation.

Reply:
I was checking for an empty textarea to let the normal GitHub esc key handling do its thing, in case they ever change that behavior.

Reply:
If that’s the case, the `textarea` selector is incorrect

Reply:
Hmm, yes, that was a copy and paste bug.  And the textarea *is* `event.target`.  Duh.  Fixed.